service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/NotificationHandler.java

```diff
@@ -310,6 +315,31 @@ public void sendEmailToPartiesAddedToCase(PartyAddedToCaseEvent event) {
             representativesServedByDigitalService, PARTY_ADDED_TO_CASE_THROUGH_DIGITAL_SERVICE_NOTIFICATION_TEMPLATE);
     }
 
+    @EventListener
+    public void sendFailedPBAPaymentEmailToLocalAuthority(FailedPBAPaymentEvent event) {
+        EventData eventData = new EventData(event);
+        Map<String, Object> parameters = failedPBAPaymentContentProvider.buildLANotificationParameters(
+            event.getApplicationType());
+
+        String email = inboxLookupService.getNotificationRecipientEmail(eventData.getCaseDetails(),
+            eventData.getLocalAuthorityCode());
+
+        notificationService.sendEmail(APPLICATION_PBA_PAYMENT_FAILED_TEMPLATE_FOR_LA, email, parameters,
+            eventData.getReference());
+    }
+
+    @EventListener
+    public void sendFailedPBAPaymentEmailToCTSC(FailedPBAPaymentEvent event) {
+        EventData eventData = new EventData(event);
+        Map<String, Object> parameters = failedPBAPaymentContentProvider.buildCtscNotificationParameters(
+            eventData.getCaseDetails(), event.getApplicationType());
+
+        String email = ctscEmailLookupConfiguration.getEmail();
```

Comment:
This will not fallback to HMCTS admin on older cases. Maybe not intended to do so. Let's check with Gail on this.

Reply:
Have messaged gail about this one

Reply:
resolving as this functionality will stay as is for this ticket